<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color: rgb(0, 0, 0);
}

QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}


</string>
   </property>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>921</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="PreliminaryPlot">
       <property name="styleSheet">
        <string notr="true">#PlotArea{
background:black
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox_Output2">
    <property name="geometry">
     <rect>
      <x>594</x>
      <y>90</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>DAC 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Output1">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>110</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>DAC 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 4</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_Input1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>22</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>DAC 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DAC 4</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Input1Name">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Click to edit name</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_Basename{
color: rgb(168, 168, 168)
}</string>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Output1Name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Click to edit name</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#lineEdit_Input1Name{
background: black;
border: none;
color: rgb(168,168,168)
}</string>
    </property>
    <property name="text">
     <string>Output 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Output2Name">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>59</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Click to edit name</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Output 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_StartSweep">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Sweep</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_MinVoltage">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_MaxVoltage">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MinVoltage">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MaxVoltage">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Numberofstep">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of step</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_Numberofstep">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>230</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lineedit_Delay">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>260</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Delay">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>260</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PushButton_done">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>dn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SQUID">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/nSOTScanner/Pictures/SQUIDRotated2.png);
background-color: rgb(3, 3, 3);</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MinVoltage_Unit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>170</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MaxVoltage_Unit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>200</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(V)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Delay_Unit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(ms)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
